main/main.go

```diff
@@ -42,17 +42,21 @@ func runCmd() error {
 		flagGCSBucketName   = flag.String("gcs-bucket-name", "", "GCS bucket name to upload CVE data to")
 		flagGCSBucketPrefix = flag.String("gcs-bucket-prefix", "", "GCS bucket prefix to upload CVE data under")
 		flagDryRun          = flag.Bool("dry-run", false, "Skip uploading CVE data to GCS")
+		flagOutputDir       = flag.String("output-dir", "", "Directory to write output when using dry-run")
 	)
 	flag.Parse()
 
 	if !*flagDryRun && *flagGCSBucketName == "" {
 		return errors.New("-gcs-bucket-name is empty and -dry-run was not set")
 	}
+	if *flagDryRun {
+		log.Info("Running in dry-run mode...")
+	}
```

Comment:
super nit:
```suggestion
	if *flagDryRun {
		log.Info("Running in dry-run mode...")
	} else if *flagGCSBucketName == "" {
		return errors.New("-gcs-bucket-name is empty and -dry-run was not set")
	}
```
I find that somewhat nicer